docs/network-mobile/mobile-mappers.mdx

```diff
@@ -17,14 +17,13 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
 
 :::tip Coming Soon
 
-The details on how data from Mobile Mappers will be used in rewards calculations are currently in
-development. Check out how MOBILE Mappers will be included in the MOBILE PoC Roadmap
-[here](https://blog.helium.com/mobile-proof-of-coverage-the-road-ahead-73a25601a13d).
+The details on how data from Mobile Mappers will be used in rewards calculations have 
+yet to be developed, with no estimated completion date. 
 
 :::
 
-Mobile Mappers are hardware devices explicitly designed to attach to CBRS Small Cell Radios to
-verify coverage quality and location in a particular hex.
+Mobile Mappers are hardware devices explicitly designed to attach to CBRS Small Cell Radios
+and Wi-Fi Access Points to verify coverage quality and location in a particular hex.
 
 Following [HIP-53](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md), Mobile Mappers will
-be eligible to earn 10% of Mobile rewards for verifying the Network.
+be eligible to earn 20% of Mobile rewards for verifying the Network.
```

Comment:
probably worth adding that this splits rewards with existing Discovery Mappers / subscribers.